azurerm/resource_arm_app_service_plan.go

```diff
@@ -191,8 +191,15 @@ func resourceArmAppServicePlanCreateUpdate(d *schema.ResourceData, meta interfac
 		appServicePlan.AppServicePlanProperties.PerSiteScaling = utils.Bool(v.(bool))
 	}
 
-	if v, exists := d.GetOkExists("reserved"); exists {
-		appServicePlan.AppServicePlanProperties.Reserved = utils.Bool(v.(bool))
+	reserved, reservedExists := d.GetOkExists("reserved")
+	if kind == "Linux" {
```

Comment:
could we make this:

```suggestion
	if strings.EqualsFold(kind, "Linux") {
```

Reply:
Sure, feel free. Wasn't aware of that method ;)